Bugzilla/DB/Pg.pm

```diff
@@ -118,7 +118,7 @@ sub sql_not_regexp {
 
     $self->bz_check_regexp($real_pattern) if !$nocheck;
 
-    return "${expr}::text !~* $pattern" 
+    return "${expr}::text !~* $pattern"
```

Comment:
The semicolon-free return pattern here might be worth a second pass in a different PR.